src/components/project/export/ProjectExport.vue

```diff
@@ -0,0 +1,201 @@
+<template>
+    <div>
+        <v-unipept-card class="mb-6">
+            <v-card-title class="text-h5 font-weight-bold mb-2">
+                🧬 Export Project Overview
+            </v-card-title>
+            <v-card-text>
+                <p>
+                    You’re about to export your entire project. This includes all samples, groups, analysis results, and settings.
+                    The export will be saved as a <strong>.zip</strong> file that can later be re-imported or shared with others.
```

Comment:
I thought we decided to go for a `.unipept`-file? (Which is essentially a ZIP in disguise). That way, users are less likely to fiddle around with these files or open them with other applications.